src/sentry/integrations/slack/webhooks/action.py

```diff
@@ -639,22 +615,21 @@ def _handle_group_actions(
         # Handle interaction actions
         for action in action_list:
             try:
-                if action.name == "status" or (
-                    use_block_kit
-                    and action.name
-                    in (
-                        "ignored:forever",
-                        "ignored:until_escalating",
-                        "unresolved:ongoing",
-                    )  # TODO: delete the first two names when block kit is GA
+                if action.name in (
+                    "status",
+                    "ignored:forever",
+                    "ignored:until_escalating",
+                    "unresolved:ongoing",
                 ):
+                    # check to see which action names are in use
+                    logger.info("slack.action.name", extra={"name": action.name})
```

Comment:
I wasn't really sure if it was safe to remove the ignored ones, so I'm adding a logger and we can check if it's ever sent.